src/components/block/InteractiveMap/MapKey.module.css

```diff
@@ -0,0 +1,22 @@
+.mapKey {
+  background-color: #fff;
+  bottom: 50px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  left: 60px;
+  padding: 30px 35px 30px 30px;
+  position: absolute;
+}
+
+.mapKeyEntry {
+  align-items: center;
+  display: flex;
+}
+
+.mapKeyEntry:not(:last-child) {
+  margin-bottom: 20px;
+}
+
+.mapKeyText {
+  font-family: var(--font-family-lato);
+  padding-left: 28px;
```

Comment:
Padding doesn't exactly feel like the "right" property to set here, since it's something that actually changes the shape and size of this element. Even though this is invisible today, since this element has neither a border nor a background, it's still something that feels conceptually a bit "wrong", and it could cause minor issues in the future if we did want to set a background, a border, or something else that cares about the actual size of the component.

One neat thing is that the [`row-gap`/`column-gap`/`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap) CSS property is actually supported in flexbox on newer browsers, but that might be too new for us, given that [Safari doesn't support it yet and Chrome and Edge only support it as of July last year](https://caniuse.com/flexbox-gap). But instead, I think you could make this whole key component a grid rather than a sequence of flex rows and get the benefit of being able to set the row and column gaps.